portal/src/pages/my/privacy.vue

```diff
@@ -146,19 +150,54 @@ export default {
     },
     submitButtonLabel() {
       if (
+        !this.$store.getters.hasGivenCommunityPermission &&
+        this.hasInitialCommunityPermission
+      ) {
+        return this.$t('Delete-account-and-login')
+      } else if (
         this.$store.getters.hasGivenCommunityPermission !==
         this.hasInitialCommunityPermission
       ) {
         return this.$t('save-privacy-settings-and-logout')
       }
       return this.$t('save-privacy-settings')
+    },
+    submitButtonStyle() {
+      if (
+        !this.$store.getters.hasGivenCommunityPermission &&
+        this.hasInitialCommunityPermission
+      ) {
+        return 'button privacy__form__button warning'
+      }
+      return 'button privacy__form__button'
     }
   },
   mounted() {
     this.hasInitialCommunityPermission = this.$store.getters.hasGivenCommunityPermission
   },
   methods: {
+    deleteAccount() {
+      this.isSubmitting = true
+      this.$store.dispatch('deleteUser').then(() => {
+        this.isSaved = true
+        setTimeout(() => {
+          this.isSaved = false
```

Comment:
is dit nog nodig? Na logout kun je niet meer op deze pagina blijven toch?

Reply:
zie boven